vcd/writer.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, file, timescale='1 us', date=None, comment='',
         self._scope_var_names = {}
         self._scope_types = {}
         self._ident_values = OrderedDict()
+        self._init_timestamp = init_timestamp
         self._prev_timestamp = 0
```

Comment:
I don't think this new `_init_timestamp` instance attribute is required. Instead, I think we can just use the new `init_timestamp` parameter to initialize the existing `_prev_timestamp` attribute.